tests/test_visitors/test_ast/test_classes/test_getter_setter.py

```diff
@@ -0,0 +1,158 @@
+# -*- coding: utf-8 -*-
+
+import pytest
+
+from wemake_python_styleguide.violations.oop import (
+    UnpythonicGetterSetterViolation,
+)
+from wemake_python_styleguide.visitors.ast.classes import WrongClassVisitor
+
+module_getter_and_setter = """
```

Comment:
We also need at least one case for `@dataclass`

Reply:
I would like to clarify whether or not the following case should be prohibited:
```
"""
@dataclass
class DataClass(object):
    attribute: int

    def get_attribute(self):
        ...

    def set_attribute(self):
        ...
"""
```
As I understand the rule, this should not be allowed?

Reply:
Yes, this should also be forbidden.

Reply:
Attributes like `attribute: int` are of type ast.Load but are they the only ones of type ast.Load? What i am asking is how do we find them? Because they are not of type attribute.

Reply:
They are `ast.AnnAssign`

Reply:
Thank you! Is there a method to print the type of an ast node for next debugging purposes?

Reply:
See https://github.com/wemake-services/wemake-python-styleguide/blob/master/CONTRIBUTING.md#helpers